tests/test_config_entries.py

```diff
@@ -3286,7 +3286,9 @@ async def test_reauth(hass):
     await entry.async_setup(hass)
     await hass.async_block_till_done()
 
-    entry.async_start_reauth(hass, {"extra_context": "some_extra_context"})
+    entry.async_start_reauth(
+        hass, context={"extra_context": "some_extra_context"}, data={"extra_data": 1234}
```

Comment:
I want to check `data` is actually passed in this test but wasn't sure how - it didn't seem to be available on `flows`. Should i be patching the flow init call or mocking out async_step_reauth or something else?

Reply:
The similar tests are patching `hass.config_entries.flow.async_init`

Reply:
Cheers. Done in d59973c525b9b97c6caa542c62d58d4c30ef05fc.